debian/patches/0004-obexd-enable-ebook-backend-conditionally.patch

```diff
@@ -1,103 +0,0 @@
-From 5acde5532f16e1bb8b354baee00f8fe71fc38e52 Mon Sep 17 00:00:00 2001
-From: Simon Fels <simon.fels@canonical.com>
-Date: Fri, 11 Sep 2015 19:01:52 +0200
-Subject: [PATCH 4/4] obexd: enable ebook backend conditionally
```

Comment:
I guess we dont need eds anymore so i guess this is fine to be dropped.

Reply:
alright!